### ISSUE TITLE
config.xml not found

### ISSUE THREAD

**Issue by pastelmind** [HeroesToolChest/HeroesDataParser#68]
Using HeroesDataParser version 4.5.1 on Windows 10. The following error appears:

```
PS C:\Users\Phil\Documents\GitHub> ~/hdp/heroesdata.exe 'C:\Program Files (x86)\Heroes of the Storm' --json -l kokr -l enus -o .

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Heroes Data Parser (4.5.1)
  --https://github.com/HeroesToolChest/HeroesDataParser
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

C:\Users\Phil\AppData\Local\Temp\.net\heroesdata\t1xj2ows.hie\config.xml not found. Unable to continue.
```

Note: Manually copying the bundled `config.xml` to the directory path above makes the program work.

**Comment by MGatner** [comment]
I had a comparable issue (and solution) on macOS - I wrote it off as a one-time fluke but maybe it wasn't.

**Comment by koliva8245** [comment]
Only happens on fde or scd releases. The global tool and fde are fine.

**Comment by MGatner** [comment]
Still an issue with 4.5.1-build.20191215.1

**Comment by koliva8245** [comment]
Try it again with the latest dev.

**Comment by MGatner** [comment]
Yes, fixed with 20191216.4
